### PR TITLE
removed date from blog URLs

### PR DESCRIPTION
**Pull request by tristandenyer**
We changed the format of the blog URLs, dropping the date from them.
This is a general cleanup of all blog URLs to remove the date.

### CODE REVIEW

File: src/pages/docs/postman/transport-layer-security.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
```
Comment (SueSmith) [inline src/pages/docs/postman/transport-layer-security.md:1]: looks like this one sneaked in lol

**Comment by arlemi** [comment]
@tristandenyer Any update here? Looks like a few conflicts came into play since your last commit 😅

File: src/pages/docs/postman/transport-layer-security.md (hunk 1 above)
Comment (tristandenyer) [reply src/pages/docs/postman/transport-layer-security.md:1]: @SueSmith Been looking over this and I am not sure what you mean by this. What am i supposed to do here? Not seeing a blog link with date.

Comment (SueSmith) [reply src/pages/docs/postman/transport-layer-security.md:1]: this file was included in the pr but shouldn't be, it isn't in the docs - I think it was from this previous pr https://github.com/postmanlabs/postman-docs/pull/2319

**Comment by tristandenyer** [comment]
Updated, should be good to review.

File: src/pages/docs/postman/transport-layer-security.md (hunk 1 above)
Comment (tristandenyer) [reply src/pages/docs/postman/transport-layer-security.md:1]: @SueSmith Appears to be deleted now.

Comment (SueSmith) [reply src/pages/docs/postman/transport-layer-security.md:1]: still showing here 🤔

File: src/pages/docs/postman/sending-api-requests/making-soap-requests.md [hunk 2]
```diff
@@ -13,8 +13,13 @@ contextual_links:
   - type: subtitle
     name: "Related Blog Posts"
   - type: link
+<<<<<<< HEAD
```
Comment (SueSmith) [inline src/pages/docs/postman/sending-api-requests/making-soap-requests.md:16]: conflict issue here

File: src/pages/docs/postman/transport-layer-security.md (hunk 1 above)
Comment (tristandenyer) [reply src/pages/docs/postman/transport-layer-security.md]: I don't see that file locally: 
Screen Shot 2020-05-22 at 3 21 39 PM

**Comment by tristandenyer** [comment]
This PR is turning into a soup sandwich—only getting worse the more it gets touched and the more time goes by. Going to delete it and redo from a fresh pull off develop.

**Pull request closed without merging**